polyglot.js

```diff
@@ -60,6 +92,7 @@ class PolyGlot {
 			}
 		}
 	}
+
```

Comment:
Adding a new line (a few) between functions. That's not a bad change in itself, and I should have better spaced my functions, but I'd prefer to have that done separately on its own commit.
I'm also reviewing this with a "ignore whitespace changes" in the diff, I would like to review it without that. I don't like having a commit do more than a single thing (i.e: fix some formatting such as the newlines here and adding wfrp support). So please fix the spaces to tab conversion you did, remove these, and squash it with your previous commits.